SwiftLint.xcodeproj/project.pbxproj

```diff
@@ -2195,7 +2247,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = D0D1212819E878CC005E4BAA /* Release.xcconfig */;
 			buildSettings = {
-				MACOSX_DEPLOYMENT_TARGET = 10.10;
+				MACOSX_DEPLOYMENT_TARGET = 10.11;
```

Comment:
had to bump this because of `BlueSocket` - hopefully, it's ok since 10.11 is already pretty old by now